signature-aggregator/aggregator/aggregator_test.go

```diff
@@ -35,23 +49,77 @@ func instantiateAggregator(t *testing.T) (
 			constants.DefaultNetworkCompressionType,
 			constants.DefaultNetworkMaximumInboundTimeout,
 		)
-		require.Equal(t, err, nil)
+		require.Equal(t, nil, err)
 	}
 	aggregator, err := NewSignatureAggregator(
 		mockNetwork,
-		logging.NoLog{},
+		logging.NewLogger(
+			"aggregator_test",
+			logging.NewWrappedCore(
+				logging.Debug,
+				os.Stdout,
+				zapcore.NewConsoleEncoder(
+					zap.NewProductionEncoderConfig(),
+				),
+			),
+		),
 		1024,
 		sigAggMetrics,
 		messageCreator,
 	)
-	require.Equal(t, err, nil)
+	require.Equal(t, nil, err)
 	return aggregator, mockNetwork
 }
 
+func makeConnectedValidators(validatorCount int) (*peers.ConnectedCanonicalValidators, []*bls.SecretKey) {
+	var validatorSet []*warp.Validator
+	var validatorSecretKeys []*bls.SecretKey
+
+	nodeValidatorIndexMap := make(map[ids.NodeID]int)
+
+	for i := 0; i < validatorCount; i++ {
+		secretKey, err := bls.NewSecretKey()
+		if err != nil {
+			panic(err)
+		}
+		validatorSecretKeys = append(validatorSecretKeys, secretKey)
+
+		pubKey := bls.PublicFromSecretKey(secretKey)
+
+		nodeID, err := ids.ToNodeID(utils.RandomBytes(20))
+		if err != nil {
+			panic(err)
+		}
+		nodeValidatorIndexMap[nodeID] = i
+
+		fmt.Printf(
+			"validator with pubKey %s has nodeID %s\n",
+			hex.EncodeToString(bls.PublicKeyToUncompressedBytes(pubKey)),
+			nodeID.String(),
+		)
+
+		validatorSet = append(validatorSet,
+			&warp.Validator{
+				PublicKey:      pubKey,
+				PublicKeyBytes: bls.PublicKeyToUncompressedBytes(pubKey),
+				Weight:         1,
+				NodeIDs:        []ids.NodeID{nodeID},
+			},
+		)
+	}
+
+	return &peers.ConnectedCanonicalValidators{
+		ConnectedWeight:       uint64(validatorCount),
+		TotalValidatorWeight:  uint64(validatorCount),
+		ValidatorSet:          validatorSet,
+		NodeValidatorIndexMap: nodeValidatorIndexMap,
+	}, validatorSecretKeys
+}
+
 func TestCreateSignedMessageFailsWithNoValidators(t *testing.T) {
 	aggregator, mockNetwork := instantiateAggregator(t)
 	msg, err := warp.NewUnsignedMessage(0, ids.Empty, []byte{})
-	require.Equal(t, err, nil)
+	require.Equal(t, nil, err)
```

Comment:
We should use `require.NoError(t, err)` here and elsewhere

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/467/commits/0c630f0c4fe97af16d2220efab508982ed4e0dfe